OpenDreamShared/Dream/ColorHelpers.cs

```diff
@@ -29,7 +29,8 @@
 
     public enum ColorSpace {
         RGB = 0,
-        HSV = 1
+        HSV = 1,
```

Comment:
L bot

Reply:
This can be fixed by adding it as an abbreviation in our [DotSettings](https://github.com/OpenDreamProject/OpenDream/blob/master/OpenDream.sln.DotSettings#L10).